### ISSUE TITLE
Overfull hbox on title page

### ISSUE THREAD

**Issue by chvp** [niknetniko/ugent2016#10]
With
```
\usepackage[type=report]{ugent2016-title}
\academicyear{2023–2024}
\subtitle{Learn to code with a data-driven platform}
\titletext{A dissertation submitted to Ghent University in partial fulfilment of the requirements for the degree of Doctor of Computer Science.}
\promotors{%
  Supervisors:\\
  Prof.\ Dr.\ Peter Dawyndt\\
  Prof.\ Dr.\ Ir.\ Bart Mesuere\\
  Prof.\ Dr.\ Bram De Wever
}
```
on a paper size of 240mm by 170mm, I'm getting an overfull hbox on the title page: `Overfull \hbox (10.95pt too wide) in paragraph at lines 45--45` (line 45 is the `\maketitle` statement).

**Comment by niknetniko** [comment]
This might be fixed by version [0.12.0](https://github.com/niknetniko/ugent2016/releases/tag/0.12.0)

**Comment by chvp** [comment]
Yeah, seems fixed, thanks!
